.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,50 @@
+name: e2e-tests
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/integration-tests.yml'
+      - '**.py'
+      - '**.ts'
+      - '**.tsx'
+      - 'package.json'
+      - 'package-lock.json'
+      - 'tsconfig.json'
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        optuna-version: ['optuna==2.10.0', 'git+https://github.com/optuna/optuna.git']
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - run: npm install
+      - run: npm run build:dev
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
```

Comment:
How about using Python 3.10?

FYI: The reason why visual regression tests uses Python 3.8 is just pyppeteer does not supprt Python 3.9 or later.

Reply:
Updated.